code/game/objects/structures/tables_racks.dm

```diff
@@ -544,11 +544,12 @@
 	check_patient()
 
 /obj/structure/table/optable/proc/check_patient()
-	var/mob/living/carbon/human/M = locate(/mob/living/carbon/human, loc)
+	var/mob/living/carbon/M = locate(/mob/living/carbon, loc)

```

Comment:
```suggestion
	var/mob/living/carbon/M = locate(/mob/living/carbon) in loc

```

Reply:
What's the difference?

Reply:
one is correct and one isnt

Reply:
there's no two argument version of locate()

Reply:
>     locate(Type) in Container 
    locate(X,Y,Z) 
    locate(Tag) 
    locate(TextRef)

Reply:
What about something like: 
`locate(/obj/machinery/computer/operating, get_step(src, direction))`?

Reply:
There's no two argument version of `locate()`